src/iris/api.py

```diff
@@ -5847,6 +5847,36 @@ def on_delete(self, req, resp, username, application):
         resp.status = HTTP_204
 
 
+class InternalRenderJinja():
```

Comment:
Will this also render the markdown syntax used in email templates?

Reply:
@jrgp no, the markdown will be rendered separately within iris-message-processor. This endpoint is exclusively to render tracking notification jinja (I still have to make an RB for that change).